src/Microsoft.SqlTools.ServiceLayer/Microsoft.SqlTools.ServiceLayer.csproj

```diff
@@ -15,7 +15,7 @@
 		<AssemblyTitle>SqlTools Editor Services Host Protocol Library</AssemblyTitle>
 		<Description>Provides message types and client/server APIs for the SqlTools Editor Services JSON protocol.</Description>
 		<!-- False alerts, disabled due to issue: https://github.com/dotnet/roslyn/issues/65850 -->
-		<NoWarn>$(NoWarn);CS8795</NoWarn>
+		<NoWarn>$(NoWarn);CS8795;SYSLIB1045</NoWarn>
```

Comment:
1. Why are you disabling this warning?
2. Whenever we disable a warning we should be adding a comment on why it's disabled (such as what is already there for the CS8795 warning)

Reply:
Moved to GeneratedRegexAttribute